### ISSUE TITLE
Reader: Show preview also for footnotes and figures

### ISSUE THREAD

**Issue by michael** [substance/texture#572]
I've implemented a preview (on mouseover) for references. @Integral when you get time you could implement a similar preview for footnotes and figures/tables.

When the xref is clicked we want to scroll to the target eventually, so if you have implementation ideas for that as well, please go ahead (but keep in a separate PR and only after the previews are implemented)
